python/ray/tune/trainable.py

```diff
@@ -466,16 +465,41 @@ def export_model(self, export_formats, export_dir=None):
         export model to local directory.
 
         Args:
-            export_formats (list): List of formats that should be exported.
+            export_formats (Union[list,str]): Format or list of (str) formats
+                that should be exported.
             export_dir (str): Optional dir to place the exported model.
                 Defaults to self.logdir.
 
         Returns:
             A dict that maps ExportFormats to successfully exported models.
         """
+        if isinstance(export_formats, str):
+            export_formats = [export_formats]
         export_dir = export_dir or self.logdir
         return self._export_model(export_formats, export_dir)
 
+    def import_model(self, import_file):
+        """Imports a model from import_file.
+
+        Subclasses should override _import_model() to actually
+        import model from the given file and format (all of which has already
+        been checked by this parent method).
+
+        Args:
+            import_file (str): The file to import the model from.
+
+        Returns:
+            A dict that maps ExportFormats to successfully exported models.
+        """
+        # Check for existence.
+        if not os.path.exists(import_file):
```

Comment:
Should this code be in rllib's Trainer instead of the tune module (cc @richardliaw )?

Reply:
Yeah I think so; the export_models code-path is not meant to be symmetric in Tune (feeding in tuned models).

Reply:
Hmm, maybe you have some pretrained weights from somewhere and want to continue training an Agent via tune?
Not sure, but it doesn't sound completely off.
But I can move it.

Reply:
done